configs/default/DREA-DreamworksF722.config

```diff
@@ -0,0 +1,138 @@
+# Betaflight / STM32F7X2 (S7X2) 4.0.0 Mar 26 2019 / 19:51:34 (173e958da) MSP API: 1.41
+
+#define USE_ACC_SPI_MPU6000
+#define USE_GYRO_SPI_MPU6000
+#define USE_ACC_SPI_ICM20689
+#define USE_GYRO_SPI_ICM20689
+#define USE_GYRO_SPI_ICM42688P
+#define USE_ACC_SPI_ICM42688P
+#define USE_BARO_BMP280
+#define USE_BARO_DPS310
+#define USE_FLASH_M25P16
+#define USE_MAX7456
+
+board_name DreamworksF722
+manufacturer_id Drea
```

Comment:
All caps.
needs to be officially accepted to https://github.com/betaflight/unified-targets/blob/master/Manufacturers.md as well.

Reply:
@Dreamworks-fpv please provide an URL where this hardware is available.

Reply:
#1177